docs/core/versions/index.md

```diff
@@ -13,89 +13,220 @@ ms.assetid: f6f684b1-1d2c-4105-8376-7c1959e23803
+.NET Core is made of [NuGet packages](../packages.md), tools, and frameworks that are distributed as a unit. Each of these platform layers can be versioned separately, enabling better agility. While there is significant versioning flexibility in that regard, we also have a desire to version the platform as a unit to make the product easier to understand.
 
-The product is in some respects unique, being described and delivered via a package manager (NuGet) as packages. While you typically acquire .NET Core as a standalone SDK, the SDK is largely a convenience experience over NuGet packages and therefore not distinct from packages. As a result, versioning is first and foremost in terms of packages and other versioning experiences follow from there.
+This article aims at clarifying how the .NET Core SDK and runtime are versioned.
 
-## Semantic Versioning
+## Summary
+
+There are lots of moving parts that version independently in .NET Core, but starting with .NET core 2.0, we'll make sure that there is an easy to understand top-level version number that everybody understands to be *the* version of ".NET Core" as a whole. The rest of this document goes into the details of the versioning of all those parts, but you shouldn't need to know all those in order to use .NET Core, or build libraries that target it.
+
+## The past
+
+While our intent has always been clarity, in the past we've been erring too far on the side of accuracy about the contents of our downloads when naming them. The file name contained both the version of the runtime and of the tooling, but without clarity about which was which. This greatly confused some of our users.
+
+## The future
+
+Starting with the 2.0 release, we'll make sure that our downloads show a single version number in their file name.
+
+We're unifying several version numbers:
+
+* The shared framework and associated runtime
+* The SDK and CLI
+* The `Microsoft.NETCore.App` metapackage
+
+This makes it easier for users to know what version of the SDK to install on their dev machines, and what the corresponding version of the shared framework should be when time comes to provision a production environment. When downloading a SDK or runtime, the version number you'll see is going to be the same.
+
+### Installers
+
+Going forward, downloads for our [daily builds](https://github.com/dotnet/core-setup#daily-builds) and [our releases](https://www.microsoft.com/net/download/core) will adhere to a new naming scheme that will be easier to understand.
+The installer UI in those downloads will also be modified to clearly present the names and versions of the components being installed. In particular, titles will show the same version number that is in the download's file name.
+
+#### File name format
+
+`[product]-[major].[minor].[patch]-[previewN]-[optional build #]-[rid]-[component].[file ext]`
```

Comment:
@bleroy File format spec needs to be updated to match our last update. 

`[product]-[component]-[major].[minor].[patch]-[previewN]-[optional build #]-[rid].[file ext]`